`build LocalRelease` fails when having VS 2017 15.6.4 / F# SDK 10.1 is installed

MSBUILD : error MSB1040: ToolsVersion is not valid. The tools version "14.0" is unrecognized. Available tools versions are "15.0", "2.0", "3.5", "4.0".

--- a/build.fsx
+++ b/build.fsx
@@ -33,9 +33,6 @@ let releaseArchiveNetCore = "bin" </> "pkgs" </> "fsautocomplete.netcore.zip"
 // Pattern specifying assemblies to be tested using NUnit
 let testAssemblies = "**/bin/*/*Tests*.dll"
 
-if Environment.OSVersion.Platform = PlatformID.Win32NT then
-  MSBuildDefaults <- {MSBuildDefaults with ToolsVersion = Some "14.0"}
-
 Target "BuildDebug" (fun _ ->
   MSBuildDebug "" "Build" ["./FsAutoComplete.sln"]
   |> Log "Build-Output: "

